catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/catalyst.dart

```diff
@@ -288,6 +299,13 @@ ThemeData _buildThemeData(
     drawerTheme: DrawerThemeData(
       backgroundColor: voicesColorScheme.elevationsOnSurfaceNeutralLv0,
     ),
+    dialogTheme: DialogTheme(
+      // TODO(damian): themed value needed. We don't have it defined yet.
```

Comment:
convention here is to put your github username in case somebody wanted to contact you